gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/editor/GravatarQuickEditorPage.kt

```diff
@@ -49,21 +65,42 @@ internal fun GravatarQuickEditorPage(
                 }
             }
         }
-        composable(QuickEditorPage.OAUTH.name, enterTransition = { EnterTransition.None }) {
+        composable(route = QuickEditorPage.OAUTH.name, enterTransition = { EnterTransition.None }) {
             OAuthPage(
                 oAuthParams = oAuthParams,
                 email = gravatarQuickEditorParams.email,
                 onAuthError = { onDismiss(GravatarQuickEditorDismissReason.OauthFailed) },
                 onAuthSuccess = { navController.navigate(QuickEditorPage.EDITOR.name) },
             )
         }
-        composable(QuickEditorPage.EDITOR.name) {
-            AvatarPicker(
-                gravatarQuickEditorParams = gravatarQuickEditorParams,
-                handleExpiredSession = true,
-                onAvatarSelected = onAvatarSelected,
-                onSessionExpired = { navController.navigate(QuickEditorPage.OAUTH.name) },
-            )
+        navigation(
+            route = QuickEditorPage.EDITOR.name,
+            startDestination = EditorNavDestinations.AVATAR_SELECTION.name,
+        ) {
+            composable(route = EditorNavDestinations.AVATAR_SELECTION.name) {
+                AvatarPicker(
+                    onAvatarSelected = onAvatarSelected,
+                    onSessionExpired = { navController.navigate(QuickEditorPage.OAUTH.name) },
+                    onAltTextTapped = {
+                        navController.navigate(
+                            route = EditorNavDestinations.ALT_TEXT.name,
+                            navOptions = navOptions {
+                                popUpTo(EditorNavDestinations.AVATAR_SELECTION.name) { saveState = true }
+                            },
+                        )
+                    },
+                    viewModel = editorViewModel,
+                )
+            }
+            composable(
+                route = EditorNavDestinations.ALT_TEXT.name,
+            ) {
+                AltTextSection(
```

Comment:
I see I wasn't consistent with the naming but I planned to add a `Page` suffix to each destination. Could we rename this to `AltTextPage`?